SConstruct

```diff
@@ -101,7 +101,17 @@ options.Add(
 options.Add(
 	"LINKFLAGS",
 	"The extra flags to pass to the C++ linker during compilation.",
-	"",
+	""
+)
+
+options.Add(
+	BoolVariable( "ASAN", "Enable ASan when compiling with clang++", False)
+)
+
+options.Add(
+	"CLANG_ROOT",
```

Comment:
This seems a bit cumbersome - can we derive it from CXX perhaps? And how come we don't need it for the Cortex PR?

Reply:
> And how come we don't need it for the Cortex PR?

I think this is because of the find QT in gaffer's SConstruct which runs a small qt application a therefore a full link.

Reply:
Ah, so if we only introduce asan into `env` _after_ we've called `conf.checkQtVersion()`, can we do away with CLANG_ROOT entirely?

Reply:
> Ah, so if we only introduce asan into env after we've called conf.checkQtVersion(), can we do away with CLANG_ROOT entirely?

Perfect. Removed CLANG_ROOT in 3d5465b